modules/cluster-logging-curator-delete-index.adoc

```diff
@@ -3,61 +3,68 @@
 // * logging/cluster-logging-curator.adoc
 
 [id="cluster-logging-curator-yaml_{context}"]
-= Deleting Curator Indices
+= Configuring Curator index deletion
```

Comment:
Just checking, I thought that deleting Curator indices was in the list of things that were not supported?

Here: https://logging-efk-3--ocpdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-maintenance-support.html